lib/Doctrine/Common/Collections/Criteria.php

```diff
@@ -109,6 +109,12 @@ public function __construct(Expression $expression = null, array $orderings = nu
         }
     }
 
+    public static function fromArray(array $criteria) {
```

Comment:
@stof There is already `public static function create()`. `createFromArray()` maybe?